demos/index.html

```diff
@@ -88,7 +89,11 @@
                             gradeNames: ["gpii.firstDiscovery.auxSchema"]
                         },
                         prefsEditor: {
-                            prefsEditorType: "gpii.firstDiscovery.firstDiscoveryEditor"
+                            store: {
```

Comment:
Should the store config actually be part of the "gpii.firstDiscovery.firstDiscoveryEditor", instead of configured in the demo?